Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -49,6 +49,7 @@ class NodejsProjectNode : CommonProjectNode, VsWebSite.VSWebSite, INodePackageMo
         internal readonly RequireCompletionCache _requireCompletionCache = new RequireCompletionCache();
         private string _intermediateOutputPath;
         private readonly Dictionary<NodejsProjectImageName, int> _imageIndexFromNameDictionary = new Dictionary<NodejsProjectImageName, int>();
+        private bool? shouldAcquireTypingsAutomatically;
 
```

Comment:
Guard this with a `#if DEV14`. Otherwise the compiler will complain about the unused variable when compiling on Dev15.